### DISCUSSION TITLE
History of reputation

### DISCUSSION THREAD

**Discussion by amjpdevp** [dailydotdev/daily#1322] [Feature Requests]
There should option to show the history of reputation so the user can estimate where he/she made mistake and how can improve it

**Comment by idoshamun** [comment]
Hi @amjpdevp,

Thanks for the suggestion. I agree that this is much needed, we will consider :)

**Comment by henriqueleite42** [comment]
+1

**Comment by subrajeet-maharana** [comment]
Users should be able to track their reputation history. The reputations they earned (like someone up-voted/down-voted their post, comments etc). And their usages (like x reputation used to restore the broken streak)

Reference: Something like Leetcode does.
![image](https://github.com/user-attachments/assets/473f0eec-998e-45a1-94f7-e0255a87a1dc)
